frontend/www/js/omegaup/arena/coursev2.ts

```diff
@@ -9,19 +9,24 @@ import * as time from '../time';
 
 import Vue from 'vue';
 import arena_Course, { Tabs } from '../components/arena/Coursev2.vue';
+import { Tabs as problemTabs } from '../components/problem/Detailsv2.vue';
```

Comment:
```suggestion
import { Tabs as ProblemTabs } from '../components/problem/Detailsv2.vue';
```

los tipos van con mayúsculas.